ecs_pod_creator.go

```diff
@@ -564,6 +564,13 @@ type ECSPodPlacementOptions struct {
 	// If the strategy is binpack, it defaults to "memory".
 	// If the strategy is random, this does not apply.
 	StrategyParameter *ECSStrategyParameter
+
+	// InstanceFilter is a set of query expressions that restrict the placement
+	// of the pod to a set of container instances in the cluster that match the
+	// query filter.
+	// Docs: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/cluster-query-language.html
+	// kim: TODO: implement
```

Comment:
flagging in case you meant to remove this line

Reply:
Good catch, removed.